### PR TITLE
Feature header navigation

### PR DESCRIPTION
**Pull request by eldani17**
## Summary

- The header was implemented with the requested background
- Navigation and return to the screen were implemented

## Screenshots

test-navigation

#### Galaxy S (or similar)
image

#### Pixel 2/3 (or similar)
image

#### Pixel 3XL (or similar)
image

## Trello Card
https://trello.com/c/nRUOAqZf/15-n-74-header

### CODE REVIEW

File: wbooks/src/app/index.tsx [hunk 1]
```diff
@@ -10,24 +10,36 @@
...
   return (
     <NavigationContainer>
       <AppStack.Navigator
-        initialRouteName="Home"
+        initialRouteName="HomeList"
         screenOptions={{
-          headerStyle: {
-            backgroundColor: white
+          headerTintColor: white,
+          // headerTitleAlign: 'center', se comenta para tener comportamiento natvio iOS/Android
```
Comment (mattgle) [inline wbooks/src/app/index.tsx]: Si esto no va a ir nunca, borralo

File: wbooks/src/app/screens/bookList/index.tsx [hunk 2]
```diff
@@ -16,8 +16,15 @@ function BookList() {
 
   const itemSeparator = () => <View style={styles.separator} />;
 
+  const handleNavigateToHome = () => {
+    navigation.navigate('Home');
```
Comment (mattgle) [inline wbooks/src/app/screens/bookList/index.tsx]: Deberias tener un objeto que se llame Routes en donde ahi van a estar todos los nombres de las rutas de las pantallas, entonces aca lo que harias seria navigation.navigate(Routes.Home);

File: wbooks/src/app/screens/bookList/index.tsx [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
-import { FlatList, ListRenderItem, SafeAreaView, View } from 'react-native';
+import { FlatList, ListRenderItem, SafeAreaView, View, TouchableOpacity, Text } from 'react-native';
 import { BOOKS_MOCK } from '@constants/mockBooks';
 import Book from '@components/book';
 import { Book as IBook } from '@interfaces/bookInterface';
 
 import styles from './styles';
 
-function BookList() {
+function BookList({ navigation }: any) {
```
Comment (mattgle) [inline wbooks/src/app/screens/bookList/index.tsx]: export the following code into a navigation interface and then use it here.

```import { RouteProp, NavigationProp, NavigationState } from '@react-navigation/native';

export interface Navigation {
  route: RouteProp<Record<string, object | undefined>, string>;
  navigation: NavigationProp<Record<string, object | undefined>, string, NavigationState, {}, {}>;
}
```

**Review by mattgle — CHANGES_REQUESTED** [review]
Great work a few comments!

File: wbooks/src/app/index.tsx (hunk 1 above)
Comment (eldani17) [reply wbooks/src/app/index.tsx]: the comment is removed

File: wbooks/src/app/screens/bookList/index.tsx (hunk 2 above)
Comment (eldani17) [reply wbooks/src/app/screens/bookList/index.tsx]: a constants file is created containing the routes

File: wbooks/src/app/screens/bookList/index.tsx (hunk 3 above)
Comment (eldani17) [reply wbooks/src/app/screens/bookList/index.tsx]: the navigation interface was created

**Review by mattgle — APPROVED** [review]
Nice work!

**Pull request merged**